person.rb

```diff
@@ -1,4 +1,5 @@
 require 'uuid'
+require_relative 'nameable'
 
 class Person
```

Comment:
- [ ] Person class should be Nameable which means the `Person` class should inherit from the `Nameable` class.